scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -38,15 +40,20 @@
 # model validations
 MODEL_VALIDATION_RESULTS_FOLDER = "validation_results"
 VALIDATION_SUMMARY = "results.json"
+SUPPORTED_INFERNCE_SKU_FILE_NAME = "supported_inference_skus.json"
+SUPPORTED_INFERNCE_SKU_FILE_PATH = Path(__file__).parent / "config" / SUPPORTED_INFERNCE_SKU_FILE_NAME

```

Comment:
I'd suggest that SUPPORTED_INFERENCE_SKU_FILE_NAME provide the path relative to the config.py file (config/supported_inference_skus.json) rather than hard coding it here.

Reply:
Done